operator-pipeline-images/tests/entrypoints/test_download_test_results.py

```diff
@@ -0,0 +1,14 @@
+from unittest.mock import MagicMock, patch
+
+from operatorcert.entrypoints import download_test_results
+
+
+@patch("operatorcert.entrypoints.download_test_results.setup_argparser")
+@patch("operatorcert.entrypoints.download_test_results.download_test_results")
+@patch("operatorcert.entrypoints.download_test_results.store_results")
+def test_main(
```

Comment:
Same here.

Reply:
Here, this is the only function in the `download_test_results.py` file, so I thought it would be better to include that test.